app/models/components/course/groups_ability_component.rb

```diff
@@ -22,6 +22,7 @@ def allow_group_manager_manage_group
   end
 
   def course_group_manager_hash
-    { group_users: { user_id: user.id, role: Course::GroupUser.roles[:manager] } }
+    { group_users: { course_user_id: user.course_user_ids,
```

Comment:
I'm not too sure, I manually tested on `rails c` and `Course::GroupUser.first.course_user` did return a valid `course_user`. Will investigate this further.